### PR TITLE
refactor: remove graphql dependency from trials

### PR DESCRIPTION
**Pull request by ybt195**
# Test Plan
- UI change:
  - [ ] add screenshots
  - [ ] React? build & check storybooks
- [ ] user-facing api change: modify documentation and examples
- [ ] user-facing api change: add the "User-facing API Change" label
- [ ] bug fix: add regression test
- [ ] bug fix: determine if there are other similar bugs in the codebase
- [ ] new feature: add test coverage for any user-facing aspects
- [ ] refactor: maintain existing code coverage

### CODE REVIEW

File: cli/determined_cli/trial.py [hunk 1]
```diff
@@ -18,46 +17,26 @@
...
     # Print information about the trial itself.
     headers = ["Experiment ID", "State", "H-Params", "Start Time", "End Time"]
     values = [
         [
-            trial.experiment_id,
-            trial.state,
-            json.dumps(trial.hparams, indent=4),
-            render.format_time(trial.start_time),
-            render.format_time(trial.end_time),
+            trial["experiment_id"],
```
Comment (sidneyw) [inline cli/determined_cli/trial.py:35]: Suggestion: I used `zip` to place the header names next to the corresponding values here https://github.com/determined-ai/determined/blob/77e045c31909e0c592fba1bf359123ee16f0c531/cli/determined_cli/checkpoint.py#L37-L52.

 I found it easier to reason able the table view and add/remove columns.

File: master/static/srv/get_trial.sql [hunk 2]
```diff
@@ -31,7 +31,7 @@ FROM
                       c.labels
                FROM checkpoints c
                WHERE c.trial_id = t.id
-                 AND c.step_id = s.id ) r3) ASCHECKPOINT,
+                 AND c.step_id = s.id ) r3) AS CHECKPOINT,
```
Comment (sidneyw) [inline master/static/srv/get_trial.sql:34]: Was this typo causing a failure before?

Comment (ybt195) [reply master/static/srv/get_trial.sql:34]: Funny enough, it didn't. It just set the name of the field to ASCHECKPOINT instead of CHECKPOINT.

Comment (sidneyw) [reply master/static/srv/get_trial.sql:34]: Nice

File: cli/determined_cli/trial.py (hunk 1 above)
Comment (ybt195) [reply cli/determined_cli/trial.py:35]: Nice suggestion, it looks way better

**Pull request merged**